generators/csharp/sdk/src/options/BaseOptionsGenerator.ts

```diff
@@ -189,4 +193,29 @@ export class BaseOptionsGenerator {
                 assertNever(literal);
         }
     }
+
+    private getQueryParametersField({ includeInitializer }: OptionArgs): csharp.Field {
+        return csharp.field({
+            access: csharp.Access.Public,
+            name: "QueryParameters",
+            type: csharp.Type.optional(csharp.Type.map(csharp.Types.string(), csharp.Types.object())),
+            summary: "Additional query parameters sent with the request.",
+            get: true,
+            init: true,
+            initializer: includeInitializer ? csharp.codeblock("null") : undefined
+        });
+    }
+
+    private getBodyPropertiesField({ includeInitializer }: OptionArgs): csharp.Field {
+        return csharp.field({
+            access: csharp.Access.Public,
+            name: "BodyProperties",
```

Comment:
I think we should prefix it with `Additional` so it's clear it's an additive operation. It'll be long name but that's okay.

Reply:
Same for QueryParameters